specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/azureFirewall.json

```diff
@@ -840,10 +854,21 @@
         },
         "destinationFqdns": {
           "type": "array",
```

Comment:
You're missing 
```
"items": {
  "type": "string"
}
```

here. When property is of type `array`, it must have definition for `items`

Reply:
This is not coming from my PR though. Will check with owner.